src/rich-log/options.hh

```diff
@@ -4,30 +4,40 @@
 
 namespace rlog
 {
-// not enum class so that LOG(info) works
-enum channel
-{
-    info,
-    warning,
-    error,
-    debug
-};
-
 struct prefix
 {
-    explicit prefix(char const* v) : value(v) {}
+    explicit constexpr prefix(char const* v) : value(v) {}
```

Comment:
this has basically no effect :D logging is a runtime activity and `constexpr` does not prevent runtime values for `v`.